<launch>
  <!-- multi_goal_loop -->
  <node pkg="dashgo_tools" name="multi_goal_loop" type="multi_goal_loop" output="screen">
    <rosparam param= "goal_list">
    <!--x, y, z, w-->
    [
      <!-- Square Navigation -->
      6.525,-0.220,-0.010,1.000,
      6.525,-0.220,0.703,0.711,

      6.440,6.231,0.705,0.709,
      6.440,6.231,1.000,0.004,

      -5.555,6.011,1.000,0.004,
      -5.555,6.011,-0.709,0.705,

      -5.540,-0.460,-0.707,0.707,
      -5.540,-0.460,0.012,1.000,

      0.100,-0.293,0.000,1.000
    ]

    <!--
    deg    = z, w
    0deg   = 0, 1
    90deg  = 0.707, 0.707
    180deg = 1, 0s
    270deg = 0.707, -0.707

    -->
    </rosparam>
  </node>
</launch>
